[BUG] helm install erro, YAML parse error on kubeblocks/templates/admission/webhookconfiguration.yaml

**Describe the bug**

```
Run helm install kubeblocks kubeblocks \
  helm install kubeblocks kubeblocks \
    --set="prometheus.enabled=true" \
    --set="grafana.enabled=true" \
    --set="dashboards.enabled=true" \
    --set="dataProtection.enableVolumeSnapshot=true" \
    --set="snapshot-controller.enabled=true"
  
  helm template kubeblocks kubeblocks --validate
  
  bash /home/runner/work/kubeblocks/kubeblocks/.github/utils/helm_image_check.sh
  shell: /usr/bin/bash -e {0}
  env:
    CR_TOKEN: ***
    CHART_RELEASER_VERSION: v1.4.1
    RELEASE_VERSION: v0.4.0-alpha.3
    K3S_VERSION: v1.[2](https://github.com/apecloud/kubeblocks/actions/runs/4215737964/jobs/7317771369#step:9:2)[3](https://github.com/apecloud/kubeblocks/actions/runs/4215737964/jobs/7317771369#step:9:3)
    GITLAB_HELM_CHARTS_PROJECT_ID: 859[4](https://github.com/apecloud/kubeblocks/actions/runs/4215737964/jobs/7317771369#step:9:4)9
Error: INSTALLATION FAILED: YAML parse error on kubeblocks/templates/admission/webhookconfiguration.yaml: error converting YAML to JSON: yaml: line 1[6](https://github.com/apecloud/kubeblocks/actions/runs/4215737964/jobs/7317771369#step:9:6): did not find expected key
Error: Process completed with exit code 1.
```

--- a/deploy/helm/templates/admission/webhookconfiguration.yaml
+++ b/deploy/helm/templates/admission/webhookconfiguration.yaml
@@ -40,54 +40,6 @@ webhooks:
     service:
       name: {{ include "kubeblocks.svcName" . }}
       namespace: {{ .Release.Namespace }}
-      path: /mutate-apps-kubeblocks-io-v1alpha1-clusterversion
-      port: {{ .Values.service.port }}
-    {{- if .Values.admissionWebhooks.createSelfSignedCert }}
-    caBundle: {{ $ca.Cert | b64enc }}
-    {{- end }}
-  failurePolicy: Fail
-  name: mclusterversion.kb.io
-  rules:
-  - apiGroups:
-    - apps.kubeblocks.io
-    apiVersions:
-    - v1alpha1
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - clusterversions
-  sideEffects: None
-- admissionReviewVersions:
-  - v1
-  clientConfig:
-    service:
-      name: {{ include "kubeblocks.svcName" . }}
-      namespace: {{ .Release.Namespace }}
-      path: /mutate-apps-kubeblocks-io-v1alpha1-cluster
-      port: {{ .Values.service.port }}
-    {{- if .Values.admissionWebhooks.createSelfSignedCert }}
-    caBundle: {{ $ca.Cert | b64enc }}
-    {{- end }}
-  failurePolicy: Fail
-  name: mcluster.kb.io
-  rules:
-  - apiGroups:
-    - apps.kubeblocks.io
-    apiVersions:
-    - v1alpha1
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - clusters
-  sideEffects: None
-- admissionReviewVersions:
-  - v1
-  clientConfig:
-    service:
-      name: {{ include "kubeblocks.svcName" . }}
-      namespace: {{ .Release.Namespace }}
       path: /mutate-apps-kubeblocks-io-v1alpha1-clusterdefinition
       port: {{ .Values.service.port }}
     {{- if .Values.admissionWebhooks.createSelfSignedCert }}
@@ -105,30 +57,6 @@ webhooks:
     - UPDATE
     resources:
     - clusterdefinitions
-  sideEffects: None
-- admissionReviewVersions:
-  - v1
-  clientConfig:
-    service:
-      name: {{ include "kubeblocks.svcName" . }}
-      namespace: {{ .Release.Namespace }}
-      path: /mutate-apps-kubeblocks-io-v1alpha1-opsrequest
-      port: {{ .Values.service.port }}
-    {{- if .Values.admissionWebhooks.createSelfSignedCert }}
-    caBundle: {{ $ca.Cert | b64enc }}
-    {{- end }}
-  failurePolicy: Fail
-  name: mopsrequest.kb.io
-  rules:
-  - apiGroups:
-    - apps.kubeblocks.io
-    apiVersions:
-    - v1alpha1
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - opsrequests
   sideEffects: None
 ---
 apiVersion: admissionregistration.k8s.io/v1

